### DISCUSSION TITLE
GIA on Big Sur - will it still work?

### DISCUSSION THREAD

**Discussion by PC-Thai** [Ascoware/get-iplayer-automator#343] [General]
Hi,

I'm looking to install Big Sur and was wondering if GIA will still work and if there is anything i need to do beforehand or after to ensure it will still work? I'm currently using Catalina 10.15.7.

Thanks for all your efforts on GIA. I use it daily and love it..

Regards, Phil

**Comment by Cessnafan152** [comment]
I'm running Big Sur 11.5 and GIA has worked on all versions since installing Big Sur
